src/daemon/default_vm_image_vault.cpp

```diff
@@ -355,9 +355,8 @@ mp::VMImage mp::DefaultVMImageVault::fetch_image(const FetchType& fetch_type, co
                 // Attempt to make a sane directory name based on the filename of the image
 
                 const auto image_dir_name =
-                    QString("%1-%2")
-                        .arg(image_filename.section(".", 0, image_filename.endsWith(".xz") ? -3 : -2))
-                        .arg(last_modified.toString("yyyyMMdd"));
+                    QString("%1-%2").arg(image_filename.section(".", 0, image_filename.endsWith(".xz") ? -3 : -2),
+                                         QLocale::c().toString(last_modified, "yyyyMMdd"));
```

Comment:
This is perfectly correct, but could you please respect the structure `QString(...).arg(...).arg(...)` in order to modify only line 360? You can do `git rebase --interactive` to avoid adding one more commit, if you prefer.

Reply:
Sure, but is that preferable? `QString(...).arg(...).arg(...)` involves the creation of two QStrings, while `QString(...).arg(..., ...)` involves only one.